src/legacy/core_plugins/vega/public/data_model/__tests__/vega_parser.js

```diff
@@ -116,8 +116,8 @@ describe('VegaParser._parseSchema', () => {
     expect(vp.spec).to.eql({ $schema: 'https://vega.github.io/schema/vega/v3.0.json' });
     expect(vp.warnings).to.have.length(1);
   });
-  it('vega', test('https://vega.github.io/schema/vega/v3.0.json', false, 0));
```

Comment:
I think instead modifying those tests, we might want instead to add new tests, since the v3.0 schema should still work. So I think we should just create a new test testing for vega 4.0. And just modify the test that checks for invalid vega schema to v5.0.

Also these test names are super bad, so maybe we could properly rename that test to: `should support v3.0 schema`, `should support v4.0 schema`, `should not support v5.0 schema`. (Why is the test for the newer Vega spec that's not yet supported even called `vega old` :D).